packages/shared/package.json

```diff
@@ -1,5 +1,10 @@
 {
   "private": true,
   "name": "shared",
-  "version": "0.0.0"
+  "version": "0.0.0",
+  "repository": {
```

Comment:
i can update the script to skip updating when `"private": true` (and there's not already a repository field).  i like the idea of updating the private package.json files that already have a repository field.